tests/run_notebook.py

```diff
@@ -0,0 +1,80 @@
+import re
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbformat import read
+
+
+def execute_notebook(notebook_path) -> None:
+    notebook = read(notebook_path, as_version=4)
+    execute_preprocessor = ExecutePreprocessor(timeout=600)
+
+    execute_preprocessor.preprocess(notebook, {"metadata": {"path": "."}})
+
+
+def add_parameters_to_cell(cell, take_screenshot):
+    # TODO: Change hardcoded values to variables
+    parameters = {
+        "coordinate_system_name": '"global"',
```

Comment:
please remove `coordinate_system_name`, this is something that should be already present in the notebook and the script should leave it intact.